apps/epic-react/src/utils/get-podcasts.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs'
+import path from 'path'
+import matter from 'gray-matter'
+import {PodcastFrontMatter} from '@/@types/mdx-podcast'
+import serializeMDX from '@skillrecordings/skill-lesson/markdown/serialize-mdx'
+
+const isPodcastDirectory = (dir: string) => {
+  return (
+    fs
+      .statSync(path.join(process.cwd(), 'src/content/podcast', dir))
+      .isDirectory() && !dir.startsWith('.')
+  ) // This will exclude .DS_Store and other hidden files/folders
```

Comment:
`.DS_Store` is in gitignore and shouldn't be committed to git as a general rule, so I don't think this is really necessary but probably fine